app/src/main/java/org/jellyfin/mobile/model/sql/entity/ServerEntity.kt

```diff
@@ -0,0 +1,24 @@
+package org.jellyfin.mobile.model.sql.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.HOSTNAME
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.TABLE_NAME
+
+@Entity(tableName = TABLE_NAME, primaryKeys = [HOSTNAME])
+data class ServerEntity(
+    @ColumnInfo(name = HOSTNAME) val hostname: String,
+    @ColumnInfo(name = USER_ID) val userId: String,
+    @ColumnInfo(name = ACCESS_TOKEN) val accessToken: String?,
+    @ColumnInfo(name = LAST_ACTIVE_TIME) val lastActiveTime: Long,
+    @ColumnInfo(name = LAN_HOSTNAME) val lanHostname: String? = null,
```

Comment:
I think `alternativeHostname` sounds better, since it doesn't imply that it's for LAN only.

Reply:
(also, order is important. Should place hostname and this one next to each other for readability)